variable/technology/technologies.yaml

```diff
@@ -1089,3 +1089,29 @@ Rate Automatic Frequency Restoration Reserve|Electricity|Wind|OffShore:
    description: Percentage of the Maximum Active Power that can be devoted to Automatic Frequency Restoration Reserve
                 for a Wind OffShore power plant
    unit: '%'
+   
+# description of Demand flexibility potentials
+   
+Maximum Active power|Electricity|Residential|Demand Flexibility:
```

Comment:
If the variable is called Maximum Active Power then it should have as unit a power (here MW), and not be a percentage; If we stich to a percentage then the name of the variable should include "rate' or sth else, or I fear it may be confusing with al other "active power" in the nomenclature